common/djangoapps/enrollment/tests/test_api.py

```diff
@@ -150,3 +152,18 @@ def test_get_course_details(self):
     def test_data_api_config_error(self):
         # Enroll in the course and verify the URL we get sent to
         api.add_enrollment(self.USERNAME, self.COURSE_ID, mode='audit')
+
+    def test_caching(self):
+        # Add a fake course enrollment information to the fake data API
```

Comment:
Minor typo in this comment, should probably be "Add fake course enrollment information to the fake data API".